components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/index.jsp

```diff
@@ -107,10 +108,10 @@
                 %>
                 <tr>
                     <td width="50%"><a
-                            href="claim-view.jsp?store=<%=UserCoreConstants.INTERNAL_DOMAIN%>&dialect=<%=claimDialectDTO[i].getDialectURI()%>"><%=claimDialectDTO[i].getDialectURI()%>
+                            href="claim-view.jsp?store=<%=Encode.forUriComponent(UserCoreConstants.INTERNAL_USERSTORE)%>&dialect=<%=Encode.forUriComponent(claimDialectDTO[i].getDialectURI())%>"><%=Encode.forHtmlContent(claimDialectDTO[i].getDialectURI())%>
                     </a></td>
                     <td width="50%"><a title="<fmt:message key='remove.claim.dialect'/>"
-                                       onclick="removeItem('<%=UserCoreConstants.INTERNAL_DOMAIN%>','<%=claimDialectDTO[i].getDialectURI()%>','<%=UserCoreConstants.DEFAULT_CARBON_DIALECT%>');return false;"
+                                       onclick="removeItem('<%=Encode.forHtmlAttribute(UserCoreConstants.INTERNAL_USERSTORE)%>','<%=Encode.forHtmlAttribute(claimDialectDTO[i].getDialectURI())%>','<%=Encode.forHtmlAttribute(UserCoreConstants.DEFAULT_CARBON_DIALECT)%>');return false;"

```

Comment:
This parameters are passed to a javascript function in javascript execution context. Thus, javascript attribute encoding should be applied not html attribute encoding